WcaOnRails/app/models/round.rb

```diff
@@ -140,13 +140,6 @@ def to_wcif
       "timeLimit" => event.can_change_time_limit? ? time_limit&.to_wcif : nil,
       "cutoff" => cutoff&.to_wcif,
       "advancementCondition" => advancement_condition&.to_wcif,
-
-      # TODO: This is here for backwards compatibility with TNoodle 0.13.4,
-      # which looks at scrambleGroupCount. We can remove this once a new
-      # version of TNoodle is released which looks at a different field.
-      # See https://github.com/thewca/worldcubeassociation.org/issues/3059.
-      "scrambleGroupCount" => self.scramble_set_count,
```

Comment:
`TNoodle 1.13.5` was released 1,5 month ago, so I believe it's safe to get rid of this (and also update the TNoodle code to no longer refer to `scrambleGroupCount`).